packages/adapters/wagmi/src/connectors/AuthConnector.ts

```diff
@@ -76,7 +77,14 @@ export function authConnector(parameters: AuthParameters) {
 
     async getProvider() {
       if (!this.provider) {
-        this.provider = W3mFrameProviderSingleton.getInstance(parameters.options.projectId)
+        this.provider = W3mFrameProviderSingleton.getInstance({
+          projectId: parameters.options.projectId,
+          onTimeout: () => {
+            AlertController.open(ErrorUtil.ALERT_ERRORS.INVALID_APP_CONFIGURATION, 'error')
+            // eslint-disable-next-line no-console
+            console.error(ErrorUtil.ALERT_ERRORS.originNotWhitelistedSocials(window.origin))
```

Comment:
Why are the console and Alert component messages are different? Cannot they be same and we could instead console.error inside of the `open` function?